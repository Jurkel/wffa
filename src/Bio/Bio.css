

.bio-small-stats h4 {
  color: #FFD08A;
  font-size: 10px;
  font-family: 'Courier';
  width: 90%;
  margin-bottom: -13px;
  background: transparent;
}

.bio-small-stats h3 {
  color: #F2F2F2;
  font-size: 13px;
  margin-bottom: 10px;
  background: transparent;
}

.bio-small-stats {
  display: flex;
  flex-direction: column;
  justify-content: space-evenly;
  padding-right: 15px;
  padding-left: 15px;
  background: transparent;
}

.bio-school {
  background: transparent;
}

.bio-weight {
  background: transparent;
}

.bio-height {
  background: transparent;
}

.bio-dob {
  background: transparent;
}

.bio-experience {
  background: transparent;
}

.bio-fav-team {
  background: transparent;
}

.bio-fav-player {
  background: transparent;
}

.bio-goals {
  background: transparent;
}

.bio-horizontal {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  background: transparent;
}

@media screen and (min-width: 1023px) {
  .bio-small-stats {
    padding: 25px;
  }
}

@media screen and (min-width: 1439px) {
  .bio-small-stats {
    padding: 45px;
  }
}