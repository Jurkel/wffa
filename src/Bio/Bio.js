import React from 'react';
import './Bio.css';

function Bio(props) {
  return (
    <div className='bio-wrapper'>
      <h2 className='player-title'>Bio</h2>
      <div className='bio-small-stats'>
        <div className='bio-horizontal'>
          <div className='bio-height'>
            <h4>Height</h4>
            <h3>{props.managerData.height}</h3>  
          </div>
          <div className='bio-weight'>
            <h4>Weight</h4>
            <h3>{props.managerData.weight} lbs</h3>
          </div>
          <div className='bio-dob'>
            <h4>DOB</h4>
            <h3>{props.managerData.dob}</h3>
          </div> 
        </div>
        <div className='bio-school'>
          <h4>School</h4>
          <h3>{props.managerData.school}</h3>
        </div>
        <div className='bio-experience'>
          <h4>Experience</h4>
          <h3>{props.managerData.experience}</h3>
        </div>
        <div className='bio-fav-team'>
          <h4>Favorite Team</h4>
          <h3>{props.managerData.favoriteteam}</h3>
        </div>
        <div className='bio-fav-player'>
          <h4>Favorite Player</h4>
          <h3>{props.managerData.favoriteplayer}</h3>
        </div>
      </div>
    </div>
  )
}

export default Bio;