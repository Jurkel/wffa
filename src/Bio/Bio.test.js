import { shallow } from "enzyme";
import React from "react";
import Bio from "./Bio";

it("renders Bio component", () => {
  const mockProps = {
    managerData: {
      height: "",
      weight: "",
      dob: "",
      school: "",
      experience: "",
      favoriteteam: "",
      favoriteplayer: "",
    },
  };

  expect(shallow(<Bio {...mockProps} />)).toMatchSnapshot();
});
