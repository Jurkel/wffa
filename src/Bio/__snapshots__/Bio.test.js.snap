// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Bio component 1`] = `ShallowWrapper {}`;
